website/client/components/groupLink.vue

```diff
@@ -0,0 +1,22 @@
+<template lang="pug">
+  b-link(
+    v-if='group',
+    @click.prevent='goToGroup'
+    ) {{group.name}}
+
+</template>
+
+<script>
+  export default {
+    props: ['group'],
+    methods: {
+      goToGroup () {
+        if (this.group.type === 'party') {
+          this.$router.push({name: 'party'});
+        } else {
+          this.$router.push({name: 'guild', params: {groupId: this.group._id}});
```

Comment:
Does this need an `else if` clause for the Tavern? URL https://habitica.com/groups/tavern, group ID is the constant TAVERN_ID